scalableminds/webknossos
# Use GET request for tracing creation

We replaced the the way new skeleton / volume tracings are created from the gallery view. Previously a form was submitted. We switched to sending a GET request instead.

### Steps to test:
- Navigate to Dashboard --> Gallery View
- Create a new skeleton / volume tracing
- Check that the correct dataset is used

- Disable dev autoLogin in `/conf/application.conf`
- Restart server
- Navigate to `/spotlight``
- Try the create tracing without being logged in. See message about login/signup and redirect. 

### Issues:
- amendment to #2017

------
